### DISCUSSION TITLE
Remove a Recursive Task

### DISCUSSION THREAD

**Discussion by rmsf** [Extended-Thunder/send-later#307] [Q&A]
This might be a very simple question but I cannot see how to remove a recursive task, i.e., a send later set upped for 2 weeks that I want to stop after one week.
I searched everywhere, including in the Outbox but I cannot find anywhere the email that I want to stop.

Thanks,
  Rui

**Comment by jph-sendlater** [comment]
Scheduled messages are stored in the account's "drafts" folder until they are ready to be sent. The schedule will be cancelled if you delete the message, or if you open it for editing.
